Gemfile

```diff
@@ -3,7 +3,7 @@ source 'https://rubygems.org'
 git_source(:github) { |repo| "https://github.com/#{repo}.git" }
 
 ruby '3.1.1'
-
+gem 'rails-controller-testing'
```

Comment:
- [ ] Please use 
```ruby
group :development, :test do
 ...
 gem 'rspec-rails'
end
```

Reply:
I am not sure if I have to put 
```
group :development, :test do
 gem 'rails-controller-testing'
 gem 'rspec-rails'
end
```

Reply:
Applied as requested and the test works. Thank you.